examples/siro_sandbox/sandbox_app.py

```diff
@@ -402,12 +408,20 @@ def depth_to_rgb(obs):
     glfw_config.title = "Sandbox App"
     glfw_config.size = (args.width, args.height)
     gui_app_wrapper = GuiApplication(glfw_config, args.target_sps)
-    driver = SandboxDriver(args, config, gui_app_wrapper.get_sim_input())
     framebuffer_size = gui_app_wrapper.get_framebuffer_size()
+
+    # instantiate driver:
+    driver = SandboxDriver(args, config, gui_app_wrapper.get_sim_input())
+
+    # instantiate renderer:
     # note this must be created after GuiApplication due to OpenGL stuff
     app_renderer = ReplayGuiAppRenderer(
         framebuffer_size.x, framebuffer_size.y, args.use_batch_renderer
     )
+    image_drawer = ImageFramebufferDrawer(max_width=1024, max_height=1024)
+    text_drawer = TextDrawer(framebuffer_size)
+    app_renderer.set_image_and_text_drawers(image_drawer, text_drawer)
```

Comment:
This bootstrap code is getting pretty complicated. We want it to be easy to use GuiApplication and related classes. Maybe ReplayGuiAppRenderer should construct image and text drawers internally in its constructor, with constructor arguments to help construct them.

Reply:
Done